commitlint.config.js

```diff
@@ -1,8 +1,24 @@
-// @noflow
+const fs = require("fs");
+
+function fetchComponentDirs() {
+  const dirs = fs
+    .readdirSync("./packages/orbit-components/src/", { withFileTypes: true })
+    .filter(dirent => dirent.isDirectory())
+    .map(dirent => dirent.name);
+
+  const hookDirs = fs
```

Comment:
in case of using globby, you would not need that additional iteration, just to add excludes :)